src/test/java/org/aerogear/kryptowire/KWSubmitStepTest.java

```diff
@@ -0,0 +1,54 @@
+package org.aerogear.kryptowire;
+
+import hudson.EnvVars;
+import hudson.slaves.EnvironmentVariablesNodeProperty;
+import org.apache.commons.io.IOUtils;
+import org.apache.http.HttpResponse;
+import org.apache.http.StatusLine;
+import org.apache.http.client.ResponseHandler;
+import org.apache.http.client.methods.HttpPost;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.junit.*;
+import org.jvnet.hudson.test.BuildWatcher;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.jenkinsci.plugins.workflow.job.*;
+import org.omg.CORBA.Environment;
+import sun.net.www.http.HttpClient;
+
+public class KWSubmitStepTest extends Mockito {
+    @ClassRule public static BuildWatcher bw = new BuildWatcher();
+
+    @Rule public JenkinsRule j = new JenkinsRule();
+
+    String endpoint = System.getenv("KW_ENDPOINT");
+    String token = System.getenv("KW_API_KEY");
+    String platform = System.getenv("KW_PLATFORM");
+    String binaryPath = System.getenv("KW_BINARY_PATH");
```

Comment:
Do we really want to have to set real values for these here? We should at least make it that these are not ran as part of the "mvn clean package" command wyt?